client/jetpack-connect/auth-logged-in-form.jsx

```diff
@@ -93,50 +104,50 @@ class LoggedInForm extends Component {
 		if ( this.props.isSSO || doAutoAuthorize ) {
 			debug( 'Authorizing automatically on component mount' );
 			this.setState( { haveAuthorized: true } );
-			return this.props.authorize( queryObject );
+			return authorize( queryObject );
 		}
 	}
 
-	componentWillReceiveProps( props ) {
-		const { redirectAfterAuth } = props;
+	componentWillReceiveProps( nextProps ) {
+		const { goBackToWpAdmin, redirectAfterAuth, retryAuth } = nextProps;
 		const {
 			siteReceived,
 			queryObject,
 			isRedirectingToWpAdmin,
 			authorizeSuccess,
 			authorizeError,
-		} = props.jetpackConnectAuthorize;
+		} = nextProps.jetpackConnectAuthorize;
 
 		// For SSO, WooCommerce Services, and JPO users, do not display plans page
 		// Instead, redirect back to admin as soon as we're connected
-		if ( props.isSSO || props.isWoo || this.isFromJpo( props ) ) {
+		if ( nextProps.isSSO || nextProps.isWoo || this.isFromJpo( nextProps ) ) {
 			if ( ! isRedirectingToWpAdmin && authorizeSuccess ) {
-				return this.props.goBackToWpAdmin( redirectAfterAuth );
+				return goBackToWpAdmin( redirectAfterAuth );
 			}
 		} else if ( siteReceived ) {
 			return this.redirect();
-		} else if ( props.isAlreadyOnSitesList && queryObject.already_authorized ) {
+		} else if ( nextProps.isAlreadyOnSitesList && queryObject.already_authorized ) {
 			return this.redirect();
 		}
 		if (
 			authorizeError &&
-			props.authAttempts < MAX_AUTH_ATTEMPTS &&
+			nextProps.authAttempts < MAX_AUTH_ATTEMPTS &&
 			! this.retryingAuth &&
-			! props.requestHasXmlrpcError() &&
-			! props.requestHasExpiredSecretError() &&
+			! nextProps.requestHasXmlrpcError() &&
+			! nextProps.requestHasExpiredSecretError() &&
 			queryObject.site
 		) {
 			// Expired secret errors, and XMLRPC errors will be resolved in `handleResolve`.
 			// Any other type of error, we will immediately and automatically retry the request as many times
 			// as controlled by MAX_AUTH_ATTEMPTS.
 			const attempts = this.props.authAttempts || 0;
 			this.retryingAuth = true;
-			return this.props.retryAuth( queryObject.site, attempts + 1 );
+			return retryAuth( queryObject.site, attempts + 1 );
```

Comment:
I updated to use the`nextProps` action rather than `this.props` for consistency. The action creators themselves should always remain the same.